src/deserialize_sequence.cc

```diff
@@ -224,7 +224,34 @@ void extract_disabled(gul14::string_view extract, Step& step)
     step.set_disabled(val);
 }
 
-} // namespace anonymous
+} // anonymous namespace
+
+LabelUid get_sequence_label_and_id_from_filename(gul14::string_view filename)
+{
+    LabelUid result;
+
+    auto opening_bracket = filename.find('[');
+    if (opening_bracket != filename.npos)
+    {
+        result.label = unescape_filename_characters(filename.substr(0, opening_bracket));
+
+        auto closing_bracket = filename.find(']', opening_bracket);
+        if (closing_bracket != filename.npos)
```

Comment:
This does not check that the closing bracket is the last character, so a `this [is a bad] example` is somehow silently accepted.

Reply:
Good point. I am not really convinced we need a regex for that, though. Will have a look at the code.

Reply:
Fixed, the closing `]` must now be at the end of the folder name.